pkg/api/v1/types.go

```diff
@@ -2529,6 +2535,10 @@ type PodStatus struct {
 	// More info: https://github.com/kubernetes/kubernetes/blob/master/docs/design/resource-qos.md
 	// +optional
 	QOSClass PodQOSClass `json:"qosClass,omitempty" protobuf:"bytes,9,rep,name=qosClass"`
+	// The priority value which is resolved by the Admission Controller from
```

Comment:
Same thing here, update with actual name of the admission controller once you write it.